### ISSUE TITLE
Instructions Too Complex

### ISSUE THREAD

**Issue by nopara73** [nopara73/WasabiResearchExperience#21]
![image](https://user-images.githubusercontent.com/9156103/74042915-52cdf800-49c0-11ea-906d-f45675bc6c52.png)

I'd prefer:

- Hangouts Link: https://1234.hangouts.com
- Fallback Link (if the above doesn't work):  https://5678.hangouts.com

**Comment by AvivMilner** [comment]
I am not sure why I am struggling to create a hangouts link. Let me try again and clean up the instructions.

**Comment by AvivMilner** [comment]
#

**Comment by AvivMilner** [comment]
#22 Should resolve this issue
